server/service/src/requisition/request_requisition/update/test.rs

```diff
@@ -4,7 +4,7 @@ mod test_update {
     use repository::{
         mock::{
             mock_draft_request_requisition_for_update_test,
-            mock_full_draft_response_requisition_for_update_test, mock_name_store_c,
+            mock_full_new_response_requisition_for_update_test, mock_name_store_c,
```

Comment:
We don't have draft status for response requisitions.

Have renamed

Reply:
This is also what drew my attention to the ordering of errors. When I changed the status to new from draft, tests would return errors of wrong status when the requisition was also the wrong type.

This is why I changed the order of error returning rather than restructure the mocks and tests.

I am thinking best order of error returning importance would be doesn't exist > wrong store > wrong type (request vs response) > wrong status.

This touches a few other places where I haven't changed the order, but I think it would be helpful to do so